pallets/pools/src/lib.rs

```diff
@@ -705,6 +805,15 @@ pub mod pallet {
 				Error::<T>::InSubmissionPeriod
 			);
 
+			// Both changes.tranches and changes.tranche_metadata
+			// have to be NoChange or Change, we don't allow to change either or
+			// ^ = XOR, !^ = negated XOR
+			ensure!(
+				!((changes.tranches == Change::NoChange)
+					^ (changes.tranche_metadata == Change::NoChange)),
+				Error::<T>::UpdatePrerequesitesNotFulfilled
```

Comment:
`UpdatePrerequesitesNotFulfilled` has a different purpose: this is thrown when a scheduled update cannot be executed/fulfilled yet. We should use a custom error for this, something like `InvalidTrancheUpdate` probably?